bokeh/charts/_chartobject.py

```diff
@@ -109,6 +109,8 @@ def __init__(self, title, xlabel, ylabel, legend,
         self.__palette = palette
         self.__xgrid = xgrid
         self.__ygrid = ygrid
+        self.doc = doc
```

Comment:
Maybe `_doc` and `_session` as argnames then? And then don't add to the docstring since they may go away.